apps/api-express/src/app/FindParsedParagraphs.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  ParsedArticleParagraphStandalone,
+  ParsedArticle,
+  ArticleParagraphBacksUpClaim,
+  ParsedArticleParagraph,
+} from '@foodmedicine/interfaces';
+import * as scholarsScraper from '@foodmedicine/scholars-scraper';
+import * as articleParser from '@foodmedicine/article-parser';
+
+export async function getQueryResult(
+  query: string
+): Promise<ParsedArticleParagraphStandalone[]> {
+  return await findCorrelatedParagraphs(query);
+}
+
+/**
+ * Get the top percentage of an array
+ * @param arr - an array sorted in descending order
+ * @param percent - percentage of items to be returned
+ */
+function getTopPercentage<T>(arr: T[], percent = 5): T[] {
+  return arr.slice(0, Math.floor((arr.length * percent) / 100));
+}
+
+async function findCorrelatedParagraphs(
+  query: string,
+  numberOfArticles = 25,
+  scoreCutOff = 10
+): Promise<ParsedArticleParagraphStandalone[]> {
+  const articleHeads = await scholarsScraper.runScholarsScraper(
+    query,
+    '',
```

Comment:
nit: What does this empty string represent?
Sometimes if a function takes some parameter that is static (e..g this '' param, or a static 'true' param for example) it can help to do a quick inline comment to clarify what its standing in for
example:
`myFunction(thisParamHasAName, /** isTestFunction= */ true)`

Reply:
Wait this should not be here. Files got moved around strangely from branch to branch. Fixing and Deleting